testing/internal/e2e/scope/scope_test.go

```diff
@@ -0,0 +1,322 @@
+package scope_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/boundary/api/authmethods"
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/boundary/testing/internal/e2e/boundary"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestKeyDestructionCli uses the boundary CLI to test key destruction.
+func TestKeyDestructionCli(t *testing.T) {
+	e2e.MaybeSkipTest(t)
+
+	output := boundary.AuthenticateCli()
+	require.NoError(t, output.Err, string(output.Stderr))
+
+	output = e2e.RunCommand(
+		"boundary", "scopes", "create",
+		"-name", "testscope",
+		"-format", "json",
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+
+	var scopeReply scopes.ScopeCreateResult
+	require.NoError(t, json.Unmarshal(output.Stdout, &scopeReply))
+	t.Cleanup(func() {
+		output = e2e.RunCommand(
+			"boundary", "scopes", "delete",
+			"-id", scopeReply.Item.Id,
+			"-format", "json",
+		)
+		require.NoError(t, output.Err, string(output.Stderr))
+	})
```

Comment:
When you rebase with main, you could change this to
```suggestion
	scopeId := boundary.CreateNewOrgCli(t)
```